docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.2"

```

Comment:
Why define a compose? That's mostly useful when several containers have to be brought up at once or in some defined order. I don't see the purpose of using compose when there's only one container.
You could chuck the run command in the "Using docker CLI" section of the readme into a script, and that would be that. Readme would be simpler as well. There would be no options to choose from.

Reply:
As we discussed, I left the compose file, but updated the readme to be more informative.